scripts/check_layer_size.sh

```diff
@@ -9,7 +9,7 @@
 
 # 7 mb size limit
 MAX_LAYER_COMPRESSED_SIZE_KB=$(expr 7 \* 1024)
-MAX_LAYER_UNCOMPRESSED_SIZE_KB=$(expr 18 \* 1024)
+MAX_LAYER_UNCOMPRESSED_SIZE_KB=$(expr 19 \* 1024)
```

Comment:
Just as an aside. What's the point in checking the layer size if we're just gonna keep upping the limit?

Reply:
Great question, I'm not sure. I guess to make sure we don't accidentally increase the size significantly without realizing?